bin/test-bpl.sh

```diff
@@ -0,0 +1,47 @@
+GOOD=""
```

Comment:
This file has been updated with comments, usage, and error handling for CLI arguments.

I wonder if this file should be moved to a different location, however, I'm not sure where else to put it currently.  I propose we address this when the bpl validation is reimplemented.